code/_compile_options.dm

```diff
@@ -4,6 +4,10 @@
 // Uncomment the following line to compile unit tests.
 // #define UNIT_TESTS
 
+// Uncomment the following line to enable Tracy profiling.
+// DO NOT DO THIS UNLESS YOU UNDERSTAND THE IMPLICATIONS. THIS IS NOT A JOKE.
+// #define ENABLE_BYOND_TRACY
+
```

Comment:
would you please update this comment to explain a little bit about what the implications are?